app/views/tools/index.html.erb

```diff
@@ -44,14 +44,14 @@
               </a>
             </article>
           </div>
-             
-             
-             
-             
-            
+
+
+
+
+
           <div class="js-masonry-item grid-s-12 grid-m-6">
             <article class="c-card">
-              <a href="http://access.i2ifacility.org/Alternative_data_sources/" target="_blank" alt="Alternative data 
+              <a href="https://access.i2ifacility.org/Alternative_data_sources/" target="_blank" alt="Alternative data
```

Comment:
these dont seem to link to the same place, but since they are anchors, we can keep them http